recipes/qcustomplot/all/conanfile.py

```diff
@@ -39,15 +40,10 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
-        # FIXME: we shouldn't have to force shared in qt, but config file
-        #        generated by conan in qt static is likely broken, or maybe env vars.
-        self.options["qt"].shared = True
 
     def requirements(self):
-        if Version(self.version) >= "2.0.0":
-            self.requires("qt/6.4.1")
-        else:
-            self.requires("qt/5.15.7")
+        self.requires("qt/[~6.4]", transitive_headers=True, transitive_libs=True)
```

Comment:
Version ranges are only allowed for a limited set of packages on CCI currently: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

Reply:
Thx, didn't know. Fixed it to 6.4.2.